Pages/Hotels/hotelsHome.js

```diff
@@ -0,0 +1,481 @@
+}
+
+async clickBathroom() {
+    await this.commands.clickWebElement(this.plusBathroom)
+}
+
+async clickNext() {
+    await this.commands.clickWebElement(this.nextButton)
+}
+
+async isPropertyDisplayed() {
+    await this.commands.clickWebElement(this.propertyLocated)
+    return await this.commands.isWebElementDisplayed(this.propertyLocated)
+}
+
+async enterAddressText(address) {
+    await this.commands.clickWebElement(this.enterAddress)
+    await this.commands.typeInWebElement(this.enterAddress, address)
+}
+
+async intoGoingToAddress(text) {
+    const autoSuggestionElements = await $$('//li[@class="typeahead-prediction-item fds-list-item"]');
+    for (const autoSuggestionElement of autoSuggestionElements) {
+      const suggestionText = await autoSuggestionElement.getText();
+      if (
+        suggestionText.toLowerCase().localeCompare(text.toLowerCase()) === 0) {
+        await autoSuggestionElement.click();
+        break
+        }
+    }
+}
+
+async isMapDisplayed() {
+    return await this.commands.isWebElementDisplayed(this.map)
+}
+
+async sixthAvenue() {
+    return await this.commands.isWebElementDisplayed(this.insideTheAuto, '1211 6th Avenue, New York, NY, USA')
+    
+}
+
+async isPinInMapDisplayed() {
+    return await this.commands.isWebElementDisplayed(this.pinInMap)
+}
+
+async pinUnderVerification() {
+    return await this.commands.isWebElementDisplayed(this.pinLocationVerify)
+}
+
+async verifyPriceLessToMore(){
+    // await this.commands.isWebElementDisplayedWithWait(this.numOfHotelsLocator);
+    const priceLocatorArr = await this.commands.findWebElements(this.priceLocator);
+    const prices = [];
+    // const hotelNum = await this.commands.findWebElements(this.numOfHotelsLocator).length;
+    for(let i = 0; i<=priceLocatorArr.length; i++){
+        const priceText = await this.commands.getTextFromWebElement(this.priceLocator);
+        const num = priceText.split('$');
+        prices.push(num[1]);
+            if(i===priceLocatorArr.length){
+                for(let i = 0; prices[i]<=prices[i+1]; i++){
+                    return true;
+                }
+            }
+    }
+}
+async verifyAllHotelsAreSameRating(){
+    const hotelNum = await this.commands.findWebElements(this.numOfHotelsLocator).length;
+    const starRatingNum = await this.commands.findWebElements(this.starRatingLocator).length;
+    if(hotelNum===starRatingNum){
+        return true;
+    }
+}
+async clickFiveStarRating(){
+    await this.commands.isWebElementDisplayedWithWait(this.button);
+    await this.commands.scrollAndClickWebElement(this.fiveStarLocator);
+    // await this.commands.clickWebElement(this.fiveStarLocator);
+        // await browser.pause(2000);
+}
+
+async selectFromSortByDropdown(value){
+    await this.commands.scrollAndFindWebElement(this.sortDropdown);
+    await this.commands.selectFromDropdown(this.sortDropdown, value);
+}
+
+
+
+
+async switchToTab() {
```

Comment:
Parameterize this func to pass whatever contains in title, then you'll be able to reuse this func and no need to create any more func to switch like L331.